### PR TITLE
Fixed: Added Gum.csproj to the solution

### PR DESCRIPTION
**Pull request by TanukiCom**
I added Gum.csproj back to the solution.
The editor will build and run like previous.

### CODE REVIEW

File: Murder.sln [hunk 1]
```diff
@@ -11,6 +11,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Bang.Generator", "bang\src\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Bang", "bang\src\Bang\Bang.csproj", "{5BB42532-EB96-40A7-9601-E538CBAB31FD}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Gum", "gum\src\Gum\Gum.csproj", "{4CBD9371-4090-49C0-82F7-CAB7DF68326E}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
```
Comment (isadorasophia) [inline Murder.sln:18]: you need to add to the GlobalSection so it builds as part of the solution:
```
{4CBD9371-4090-49C0-82F7-CAB7DF68326E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
{4CBD9371-4090-49C0-82F7-CAB7DF68326E}.Debug|Any CPU.Build.0 = Debug|Any CPU
{4CBD9371-4090-49C0-82F7-CAB7DF68326E}.Release|Any CPU.ActiveCfg = Release|Any CPU
{4CBD9371-4090-49C0-82F7-CAB7DF68326E}.Release|Any CPU.Build.0 = Release|Any CPU
```

(not sure why the "Add existing project..." gesture to the solution didn't pick that up)

**Review by isadorasophia — CHANGES_REQUESTED** [review]
I'm mostly curious, how was the issue coming up? since *technically* Murder.Editor.csproj had a reference to the relative path of Gum

**Comment by PadTrick** [comment]
Had the same issue, there was an error during 1st opening the solution with vs_community_2022. Readding Gum to the Project solved it.
On Archlinux opening the solution with vs_code doesnt had this issue.

**Comment by isadorasophia** [comment]
I ended up adding the Gum.csproj so it doesn't block anyone, thanks for the fix though! o/

**Pull request closed without merging**